regcore/db/django_models.py

```diff
@@ -40,9 +40,16 @@ def _transform(self, reg, version):
 
     def bulk_put(self, regs, version, root_label):
         """Store all reg objects"""
-        # This does not handle subparts. Ignoring that for now
-        Regulation.objects.filter(version=version,
-                                  label_string__startswith=root_label).delete()
+        # Delete any existing regulation objects for this version and
+        # root label.
+        # XXX: This does not handle subparts. Ignoring that for now
```

Comment:
Fair enough. I _think_ everything is covered, barring the prefix problem mentioned below